README.md

```diff
@@ -1,24 +1,78 @@
 # workflow.scenario.preparation
 
 <!-- badges: start -->
-
 [![Lifecycle:
 experimental](https://img.shields.io/badge/lifecycle-experimental-orange.svg)](https://lifecycle.r-lib.org/articles/stages.html#experimental) 
 <!-- badges: end -->
 
-The goal of `workflow.scenario.preparation` is to prepare all input scenario datasets required as input by both the [pactaverse](https://rmi-pacta.github.io/pactaverse/) R packages (for investors), and [r2dii](https://rmi-pacta.github.io/r2dii.analysis/) R packages (for banks).
+Welcome to `workflow.scenario.preparation`! This tool is designed to streamline the preparation of input scenario datasets for use in either [workflow.data.preparation](https://github.com/RMI-PACTA/workflow.data.preparation) or the [r2dii](https://rmi-pacta.github.io/r2dii.analysis/) R packages.
```

Comment:
maybe a more complete description would be to add that it prepares scenario data for workflow.x (P4I use cases) and packages.y (P4B use cases). thinking about P4S here and how users of P4S workflows do not necessarily have much experience with the individual r2dii R Packages.

Reply:
yeah, I mean my real goal here is that we use this workflow.* to align the scenario inputs (for P4I OR banks), so that we just do this once and don't worry about it again.

but out of scope for this PR